src/Workspaces/CSharpTest/CodeGeneration/SyntaxGeneratorTests.cs

```diff
@@ -2736,6 +2736,7 @@ public void TestAddMembers()
             AssertMemberNamesEqual("m", Generator.AddMembers(Generator.ClassDeclaration("d"), [Generator.MethodDeclaration("m")]));
             AssertMemberNamesEqual("m", Generator.AddMembers(Generator.StructDeclaration("s"), [Generator.MethodDeclaration("m")]));
             AssertMemberNamesEqual("m", Generator.AddMembers(Generator.InterfaceDeclaration("i"), [Generator.MethodDeclaration("m")]));
+            AssertMemberNamesEqual("", Generator.AddMembers(Generator.InterfaceDeclaration("i"), [Generator.OperatorDeclaration(OperatorKind.Addition)]));
```

Comment:
i'm not seeing any tests that actually show what was actually generated.

Reply:
There doesn't seem to be much in this file in general and in this test.

Is it fine that I call VerifySyntaxRaw just for this? I didn't added them originally because it seemed out of place. I thought the goal of this test excluded sources since I didn't see any.

Reply:
There are test files for Syntax generator that do validate the source generated.  Can you add tests to those?

Reply:
There I added one for the operator member in interface